app/src/main/java/org/sopt/study/mvvmstudy/AddActivity.kt

```diff
@@ -0,0 +1,51 @@
+package org.sopt.study.mvvmstudy
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import androidx.lifecycle.ViewModelProviders
+import org.sopt.study.mvvmstudy.data.Contact
+import org.sopt.study.mvvmstudy.data.ContactViewModel
+import org.sopt.study.mvvmstudy.databinding.ActivityAddBinding
+
+class AddActivity : AppCompatActivity() {
+
+    private lateinit var contactViewModel: ContactViewModel
+    private lateinit var binding : ActivityAddBinding
+    private var id: Long? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityAddBinding.inflate(layoutInflater)
+
+        contactViewModel = ViewModelProviders.of(this).get(ContactViewModel::class.java)
+
+        if(intent!=null && intent.hasExtra(EXTRA_CONTACT_NAME) && intent.hasExtra(EXTRA_CONTACT_NUMBER) && intent.hasExtra(
+                EXTRA_CONTACT_ID)){
+            binding.etName.setText(intent.getStringExtra(EXTRA_CONTACT_NAME))
+            binding.etNumber.setText(intent.getStringExtra(EXTRA_CONTACT_NUMBER))
```

Comment:
setText 보다는 코틀린 프로퍼티 특성을 활용해서

```suggestion
            binding.etName.text = intent.getStringExtra(EXTRA_CONTACT_NAME)
            binding.etNumber.text = intent.getStringExtra(EXTRA_CONTACT_NUMBER)
```
이게 좀 더 간결할 것 같습니당~!!